ign-gui4.yaml

```
@@ -18,7 +18,7 @@ repositories:
   ign-msgs:
     type: git
     url: https://github.com/ignitionrobotics/ign-msgs
-    version: ign-msgs5
+    version: ign-msgs6
```

```suggestion
    version: master
```

And also need to update transport